src/view/draggable/connected-draggable.js

```diff
@@ -216,6 +222,11 @@ export const makeSelector = (): Selector => {
         };
       }
 
+      // a lift is in progress - do not let anything start a lift
```

Comment:
This is unrelated but I thought of adding it @jaredcrowe as it is a bit safer. It does cause another render in the lift cycle which is lame - but I think it is safest to do this